### PR TITLE
more sophisticated versioning (take 2)

### LINKED ISSUE #66: add __version__.py
in order to "properly" split up `dbt-sqlserver` and `dbt-synapse` we need to add a `__version__.py`
https://github.com/dbt-msft/dbt-synapse/pull/32

### PR DESCRIPTION
**Pull request by swanderz**
to resolve #66. should merge #62 first probably so we can get an auto-check that this PR doesn't break anything

### CODE REVIEW

**Review by jtcohen6 — APPROVED** [review]
This is the right idea. I'm still not sure why `dbt --version` isn't working:
```
$ dbt --version
installed version: 0.18.1
   latest version: 0.18.1

Up to date!

Plugins:
```

**Comment by mikaelene** [comment]
I don't understand why the CI job fails? I have merged #62 to master and it runs well.

**Comment by swanderz** [comment]
> I don't understand why the CI job fails? I have merged #62 to master and it runs well.
So I botched the copy and paste.

File: dbt/adapters/sqlserver/__version__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+version = '0.18.0b2'
```
Comment (swanderz) [inline dbt/adapters/sqlserver/__version__.py:1]: @mikaelene @jtcohen6 any guidance on what this version should be? We'll prolly cut a release soon-ish that incorporates all of these recent PRs.... 

do we include the most recent patch release (`.1`) number? what does the `b` mean?
```suggestion
version = '0.18.1b0'
```

Comment (mikaelene) [reply dbt/adapters/sqlserver/__version__.py:1]: I have not done any 0.18.1 release yet. If we fix the snapshot-check-strategy and make sure all connections variations work. Why not 0.18.1? 😀. That is a dbt version that it works well with

**Review by mikaelene — APPROVED** [review]

**Pull request merged**